externalParser

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2013-2014 K Team. All Rights Reserved.
```

Comment:
Have you tried this script on all three platforms?

Also, it looks quite unprofessional from a software engineering perspective to organize your project to have java files and parsing scripts in the top folder of your repository.  Create a new folder, `syntax`, and put everything related to syntax there, including the current `llvm-syntax.k` file which is under `semantics`.

Moreover, in terms of external parsing, instead of script `externalParser`, I would modify `DeleteLLVMComments.java` to output the resulting LLVM file to standard output and then pipe it with kast manually, with a short explanation in a README.md file under the syntax folder.  Something like this:

---

Currently, the K parser cannot parse LLVM comments.  They need to be
removed from your LLVM program before you call the `kast` tool on them.
For your convenience, we provide a `java` program that does that, called
`DeleteLLVMComments`:

    $ java DeleteLVMComments myprogram.ll

will output the resulting LLVM program to standard output, while

    $ java DeleteLVMComments myprogram.ll | kast -e

will forward it to the `kast` tool.  Therefore, if your LLVM program
contains comments, make sure that you use the pipe command
above as external parser to the `krun` tool when you execute your
program with the LLVM semantics.